sdk/spring/spring-integration-azure-core/src/main/java/com/azure/spring/integration/core/handler/DefaultMessageHandler.java

```diff
@@ -78,12 +78,12 @@ protected void onInit() {
 
     @Override
     protected void handleMessageInternal(Message<?> message) {
-        String destination = toDestination(message);
+        String dest = toDestination(message);
```

Comment:
local var "destination" hide field name will bring reading confusion. Use "dest" here for clarity and shortness.

Besides, Does "getDesitnation(message)" looks more reasonable.